k8s/namespaces/ia/case-api/demo.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ia-case-api
+  namespace: ia
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:wa-proto-demo-*
```

Comment:
this is not a valid glob. Also patch should only have the things you want apply on top of original manifest.

Reply:
Thanks, @adusumillipraveen, for your feedback. I have fixed those issues. Please, could you confirm it is OK now? Thanks. 
One more thing, please, DO NOT MERGE since some other team is using Demo and I don't want to deploy my changes just yet. Thanks.